config/fixes/convention-eccodes.yaml

```diff
@@ -359,7 +370,6 @@ convention_name:
           - msnlwrf
           - str
           - STR # AQUA ERA5
```

Comment:
why you erased rlus?

Reply:
same reason as above: it would have a wrong sign!